apps/etterlatte-behandling/src/main/kotlin/behandling/BehandlingFactory.kt

```diff
@@ -162,10 +165,61 @@ class BehandlingFactory(
         persongalleri: Persongalleri,
         mottattDato: String?,
         kilde: Vedtaksloesning,
+        request: OpprettBehandlingRequest = hentDataForOpprettBehandling(sakId),
         prosessType: Prosesstype = Prosesstype.MANUELL,
     ): BehandlingOgOppgave? {
         logger.info("Starter behandling i sak $sakId")
 
+        with(request) {
+            return if (iverksatteEllerAttesterteBehandlinger.isNotEmpty()) {
```

Comment:
Dette begynner å ligne måten ting er satt opp i brev. Godt mulig det bare er jeg som synes det er utrolig tungt å lese, men jeg er hvert fall ikke fan av det. Funksjonskall som default-verdi og mega-blokk med `with` som gjør det vanskelig å se hva som kommer hvorfra.

Reply:
Både with og default-verdien her var litt latskap for å ikkje måtte gjera så store endringar, så tok dei bort no.